README.md

```diff
@@ -66,18 +103,71 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
-<!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+### Prerequisites
+
+In order to run this project you need:
+
+
+Example command:
+
+```sh
+ npm install
+```
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd Portfolio
+  git clone git@github.com:mm-pazouki/Portfolio.git
+```
+
+
+### Install
 
-[//]: # (- [Live Demo]&#40;https://google.com&#41;)
+Install this project with:
 
 
+Example command:
+
+```sh
+  cd Portfolio
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails server
+```
```

Comment:
- [ ] [OPTIONAL] Great work implementing the changes requested inside your README file 💯. However, please kindly ensure that you give the correct instructions to run your project. Since this is not a rails application, I would kindly suggest you use more appropriate instructions here like Open using Live Server 🤓.